src/targets/gpu/kernels/include/migraphx/kernels/pooling.hpp

```diff
@@ -133,9 +133,8 @@ struct window
         auto win_start = generate_array<diff_int>(rank{}, [&](auto j) {
             diff_int w   = win[j];
             diff_int dim = i[j];
+            (void)w;
```

Comment:
instead of adding `(void)(w)` it might be better to do 
`MIGRAPHX_ASSERT(win[j] >= 1)`

Reply:
In some build, where assert is not active, the void statement is necessary-- for compilation. Adding another assert won't help in that direction.

Reply:
Umang is saying to remove the variable instead of using a void statement.